src/app/static/src/app/store/translations/locales.ts

```diff
@@ -428,7 +432,7 @@ const en: Translations = {
     enterValidEmail: "Please enter a valid email address",
     enterKey: "Enter key",
     enterPassword: "Enter a new password",
-    enterProjectName: "Please enter a name for the new project",
+    enterProjectName: "What would you like to name the project?",
```

Comment:
I think I prefer the previous text. I think it it makes clearer that it is an action the user has to take.

Reply:
I have added text according to the mockup. We can add another ticket to tackle this suggestion.

Reply:
https://github.com/mrc-ide/hint/pull/718 
done here